### DISCUSSION TITLE
Execute on new images

### DISCUSSION THREAD

**Discussion by ben-williams-ai** [pksvision/Deep-WaveNet-Underwater-Image-Restoration#11] [Q&A]
I'm curious as to whether the trained model could be applied to a new dataset of images. Would this likely work? And are there any requirements from these images, e.g a certain resolutioon. 

If this can be done is it as simple as replacing the png's in the test folder with our own and following the colab demo? https://github.com/pksvision/Deep-WaveNet-Underwater-Image-Restoration/blob/main/DeepWaveNet_demo.ipynb

**Comment by pksvision** [comment]
Hi BenUCL, 

sorry for the late reply. 

I dont think, there should be any problem with trying arbitrary sized images. It can be directly applied to any underwater image. Please try and do let me know if it works. I should be happy to help further.
